CMakeLists.txt

```diff
@@ -23,16 +23,17 @@ endif ()
 # Set project name, version, and language. Language needs to be set for compiler checks
 project(timescaledb VERSION ${VERSION} LANGUAGES C)
 
+if (NOT CMAKE_BUILD_TYPE)
+  # Default to Release builds
+  set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel" FORCE)
+endif ()
+
 message(STATUS "TimescaleDB version ${PROJECT_VERSION_MOD}. Can be updated from version ${UPDATE_FROM_VERSION}")
 message(STATUS "Build type is ${CMAKE_BUILD_TYPE}")
 
 set(PROJECT_INSTALL_METHOD source CACHE STRING "Specify what install platform this binary
 is built for")
-
-if (NOT CMAKE_BUILD_TYPE)
-  # Default to Release builds
-  set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel" FORCE)
-endif ()
+message(STATUS "Install method is '${PROJECT_INSTALL_METHOD}'")
```

Comment:
Do you need single quotes around the var? Asking just for consistency with other status messages.

Reply:
I don't, just thought it would be easier for me to pick out when looking for it. Will change

Reply:
I don't feel strongly, so you can leave if you want.